docs/generative-ai/openshift-ai-observability-summarizer.md

```diff
@@ -0,0 +1,353 @@
+- **LLM-Generated Summaries**: Use fine-tuned Llama models to create contextual insights
+- **Interactive Chat Interface**: Ask natural language questions about your metrics
+- **PromQL Query Generation**: Get specific queries and insights from conversational input
+- **Multi-Model Support**: Support for both internal and external LLM models
+- **Context-Aware Analysis**: AI assistant with full metrics context for troubleshooting
+
+### **4. Advanced Reporting System**
+- **Multi-Format Export**: Generate reports in HTML, PDF, or Markdown formats
+- **Time-Series Visualization**: Interactive charts and metric visualizations
+- **Automated Calculations**: Trend analysis and metric calculations
+- **Customizable Reports**: Tailored reports for different stakeholders and use cases
+
+### **5. Alerting & Notifications**
+- **Smart Alerting**: Set up alerts for vLLM models and OpenShift metrics
+- **Slack Integration**: Real-time notifications with contextual summaries
+- **Custom Thresholds**: Configurable alert conditions and triggers
+- **Proactive Monitoring**: Early warning system for potential issues
+
+### **6. GPU Monitoring Excellence**
+- **DCGM Metrics Support**: Automatic discovery and monitoring of:
+  - **Temperature**: GPU core and memory temperature (°C)
+  - **Power**: Real-time power consumption (Watts)
+  - **Memory**: GPU memory usage (GB) and utilization (%)
+  - **Energy**: Total energy consumption (Joules)
+  - **Performance**: GPU utilization, clock speeds (MHz)
+- **Fleet View**: Monitor GPU health across entire OpenShift cluster
+- **Vendor Detection**: Automatic identification of GPU models and vendors
+
+---
+
+## Architecture
+
+### Core Components
+
+- **llm-service**: LLM inference (Llama models)
+- **llama-stack**: Backend API for LLM operations
+- **vLLM**: Model serving with Prometheus metrics export
+- **Prometheus/Thanos**: Metrics scraping and long-term storage
+- **DCGM**: GPU monitoring and metrics collection
+- **Streamlit UI**: Multi-dashboard frontend interface
+- **MCP (Model Context Protocol)**: Metric Collection & Processing backend
+- **Report Generator**: PDF/HTML/Markdown export capabilities
+- **Alerting Service**: Real-time notifications and alert management
+
+![Architecture](img/aiobs-architecture.jpg)
+
+### **Key Dashboards**
+1. **vLLM Dashboard**: Monitor model performance, GPU usage, latency, and throughput
+2. **OpenShift Dashboard**: Fleet monitoring with cluster-wide and namespace views
+3. **Chat Interface**: Interactive Q&A with metrics-aware AI assistant
+4. **Report Generator**: Automated analysis reports in multiple formats
+
+---
+
+## Getting Started
+
+### Prerequisites
+
+- OpenShift cluster with GPU nodes (for DCGM metrics)
+- `oc` CLI with cluster-admin permissions
+- `helm` v3.x
+- `yq` (YAML processor)
+- Deployed Prometheus/Thanos
+- (Optional) DCGM exporter for GPU monitoring
+- (Optional) Slack Webhook URL for alerting
+
+### Installation
```

Comment:
I wonder if instead of duplicating all of this, it would not be easier to point people directly to the installation instructions and the repo. This will be easier for maintenance, only one place to change things related to code.
So for the Getting Started section, I would simply explain how to get to the repo, add a few words to explain that contributions are welcome, and that's it.